apps/search/api/serializers.py

```diff
@@ -27,3 +22,24 @@ def get_meta(self, model) -> Optional[dict]:
             if key in meta:
                 del meta[key]
         return model.meta.to_dict()
+
+
+class DrfSearchableModelSerializer(DrfModelSerializer):
+    """Base serializer for searchable models."""
+
+    meta = serializers.SerializerMethodField(required=False)
+
+    def get_meta(self, model) -> Optional[dict]:
+        if not hasattr(model, 'meta'):
+            return None
+        meta = model.meta.to_dict()
+        for key in ELASTICSEARCH_META_FIELDS_TO_CLEAN:
+            # TODO: find out why in some cases it's already cleaned
+            if key in meta:
+                del meta[key]
+        return model.meta.to_dict()
```

Comment:
Not sure if I'm missing something here, but maybe the logic is broken? Assuming `to_dict()` is just returning a dictionary representation of `model.meta`, is it possible that deleting keys on that dictionary is not mutating `model.meta`? Then we just return `to_dict()` again, so maybe nothing has been modified. Do we want to `meta` instead of calling `to_dict` again?